### PR TITLE
feat: [IDP-2562] :  Added the docs on UI entity picker

### PR DESCRIPTION
**Pull request by Debanitrkl**
# Harness Developer Pull Request
Added the docs on UI entity picker

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: docs/internal-developer-portal/flows/custom-actions.md [hunk 1]
```diff
@@ -263,6 +469,9 @@ properties:
             - 'project_name'          
 ```
 
+
+This custom extension is used to
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/flows/custom-actions.md:473]: incomplete?

File: docs/internal-developer-portal/flows/custom-actions.md [hunk 2]
```diff
@@ -210,13 +417,12 @@ spec:
            ui:field: HarnessProjectPicker
 ```
 
-3. `HarnessAutoOrgPicker` : It auto populates org id on project selection. So now when you select an project id as an input the org id gets selected automatically if required as an input.
-
+### 3. `HarnessAutoOrgPicker` 
 
+It auto populates org id on project selection. So now when you select an project id as an input the org id gets selected automatically if required as an input.
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/flows/custom-actions.md:422]:
```suggestion
It auto populates org ID on project selection. So now when you select an project ID as an input the org ID gets selected automatically if required as an input.
```

File: docs/internal-developer-portal/flows/custom-extensions.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
+---
+title: Supported Custom Extensions
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/flows/custom-extensions.md:2]:
```suggestion
title: Supported Workflow UI Pickers (Field Extensions)
```

File: docs/internal-developer-portal/flows/custom-extensions.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+description: These Custom Extensions are supported in Harness IDP
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/flows/custom-extensions.md:3]: "Custom Extensions" don't mean much :)

```suggestion
description: This page lists all the Workflow UI Field Extensions that are supported in Harness IDP, which can be used as the Workflow form UI pickers.
```

File: docs/internal-developer-portal/flows/custom-extensions.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+sidebar_position: 5
+---
+
+## Custom Field Extensions
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/flows/custom-extensions.md:7]:
```suggestion
## Workflow UI Field Extensions
```

File: docs/internal-developer-portal/flows/custom-extensions.md [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+
+Collecting input from the user is a very large part of the scaffolding process and Software Templates as a whole. Sometimes the built in components and fields just aren't good enough, and sometimes you want to enrich the form that the users sees with better inputs that fit better.
+
+This is where Custom Field Extensions come in.
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/flows/custom-extensions.md:11]:
```suggestion
This is where Workflow UI Field Extensions come in.
```

File: docs/internal-developer-portal/flows/custom-extensions.md [hunk 7]
```diff
@@ -0,0 +1,324 @@
...
+
+### OwnerPicker
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/flows/custom-extensions.md:13]:
```suggestion
### OwnerPicker

`OwnerPicker` is used for developers to pick a User Group from the list of Groups that exist in the account.
```

**Review by OrkoHunter — COMMENTED** [review]
`EntityPicker` should also be there along with the `OwnerPicker`.

**Review by jenil-harness — APPROVED** [review]
LGTM

**Pull request merged**